mobile/lib/features/trade/trade_screen.dart

```diff
@@ -282,7 +298,28 @@ class TradeScreen extends StatelessWidget {
 
   Widget createSubmitWidget(PendingOrder pendingOrder, Amount pnl,
       SubmitOrderChangeNotifier submitOrderChangeNotifier, BuildContext context) {
-    Widget body = Column(
+    String bottomText;
+
+    switch (pendingOrder.state) {
+      case PendingOrderState.submittedSuccessfully:
+      case PendingOrderState.submitting:
+        bottomText = "Please wait while the order is being processed. Do not close the app!";
```

Comment:
nit: Would be great if the "Do not clsoe the app!" part is more prominent on the popup. e.g. bold and in the next line?

No need to action!